fmriprep/workflows/bold/util.py

```diff
@@ -34,7 +34,8 @@
 
 
 def init_bold_reference_wf(omp_nthreads, bold_file=None, pre_mask=False,
-                           name='bold_reference_wf', gen_report=False):
+                           name='bold_reference_wf', gen_report=False,
+                           dummy_scans=None):
```

Comment:
Missing argument in docstring. BTW, please remove the `enhance_t2` which is outdated.

Reply:
Thanks! I will address this in my next round of commits